backend/src/services/friendNotification.ts

```diff
@@ -1,25 +1,44 @@
-export const friendRequestNotification = {
-  "notification": {
-    "title": "Boop Friend Request!",
-    "body": "You got a friend request on Boop",
-    // TODO add an icon that works even when the app is closed
-    "silent": false,
-    "actions": [{
-      "action": "show_friends_page",
-      "title": "View friend request"
-    }]
-  }
-};
 
-export const friendAcceptNotification = {
-  "notification": {
-    "title": "Boop Friend Request!",
-    "body": "Someone accepted your friend request, check out who that was",
-    // TODO add an icon that works even when the app is closed
-    "silent": false,
-    "actions": [{
-      "action": "show_friends_page",
-      "title": "View Friends"
-    }]
+import { database } from "../services/database";
+import { throwBoopError } from "../util/handleAsync";
+
+
+async function queryFullName(fromUUID: string): Promise<string> {
```

Comment:
`fromUUID` is a confusing variable name, since this function is also called with `toUUID` on line 32. maybe rename this parameter to `uuid` or `userUUID`?